plugins/org.komodo.shell-api/src/org/komodo/shell/api/KomodoObjectLabelProvider.java

```diff
@@ -148,5 +152,19 @@
      *        the workspace status (never <code>null</code>)
      */
     void setWorkspaceStatus( final WorkspaceStatus status );
+    
+	 /**
+     * @return Returns a node names which should be omitted from display path
+     */
+	List<String> getGroupingNodes();
```

Comment:
This collection is being used to skip over those path segments with matching names (last segments). Maybe we should call the method something like `skippedPathSegmentNames()`. Wdyt?